index.html

```diff
@@ -0,0 +1,179 @@
+                Whether you’re new to coding, adding more skills, or advancing your career, 10 hours a week will
+                prepare you for your ideal developer job.
+            </p>
+        </div>
+        <div class="title-date">
+            <h3>2021.10.22(FRI) ~ 25(MON)</h3>
+            <p>@ National Technology Center of Turkey</p>
+        </div>
+    </section>
+    <section class="home-program">
+        <div class="program-title">
+            <h4>Main Program</h4>
+            <hr>
+        </div>
+        <ul class="program-list">
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-school"></i>
+                    <h4>Lecture</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-desktop"></i>
+                    <h4>Online Presentation</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div>
+                    <i class="fas fa-certificate"></i>
+                    <h4>Certificate</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-stamp"></i>
+                    <h4>Job Guarantee</h4>
+                </div>
+
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+            <li class="program-list-card">
+                <div class="program-list-text">
+                    <i class="fas fa-chalkboard-teacher"></i>
+                    <h4>Strong Team</h4>
+                </div>
+                <p>Listen to the speakers from various countries about the messeges of sharing and opening</p>
+            </li>
+        </ul>
+        <div class="program-paragraph">
+            <a href="#">Join CC Global Summit 2015</a>
+        </div>
+        <div class="program-paragraph-desktop">
+            <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+    <div class="program-speakers-title">
+        <h2>Featured Speakers</h2>
+        <hr>
+    </div>
+
+    <section class="program-speakers">
+
+ 
+       
+
+    </section>
+    <section class="program-speakers">
+        <div class="speakers-more">
+            <button id="loadMore"> More <i class="fas fa-arrow-down"></i></button>
+        </div>
+    </section>
```

Comment:
- [ ] This section of articles is organized in two columns of three articles each. Currently, you only have two articles on the right column, let's add one more.

![Screenshot from 2021-10-20 14-14-34](https://user-images.githubusercontent.com/40503704/138099925-6a7eb759-1cdb-4d49-a9eb-18619e47cc43.png)

- [ ] After clicking on the view more button on mobile size view and going back to desktop size view there are more than the expected articles in the section. Let's fix that in your js function

![Screenshot from 2021-10-20 14-07-43](https://user-images.githubusercontent.com/40503704/138099992-27043196-2491-4efa-a91c-42f3803c6099.png)